.gitignore

```diff
@@ -1,2 +1,3 @@
 *.DS_Store
 .swiftpm/
+*.xcuserstate
```

Comment:
maybe you want to take a look to https://www.toptal.com/developers/gitignore 
You can enter `Xcode` and it will give you a ready-to-use `.gitignroe`

Reply:
Update `.gitignore` in [3a89517](https://github.com/open-turo/token-text-view/pull/15/commits/3a895174ca40d28a8d96e0e76d4e2eabde14a802)